src/videojs.ads.js

```diff
@@ -430,9 +429,28 @@ var
         if (player.ads.state !== 'ad-playback') {
           player.trigger('adskip');
         }
+      },
+
+      stitchedAds: function(arg) {
+        if (arg) {
```

Comment:
Should this be `arg !== undefined`? It doesn't look like there would be a way to set it to `false`.